app/components/post_list/post/body/message/message.tsx

```diff
@@ -94,7 +94,7 @@ const Message = ({currentUser, highlight, isEdited, isPendingOrFailed, isReplyPo
                             layoutWidth={layoutWidth}
                             location={location}
                             postId={post.id}
-                            textStyles={textStyles}
+                            textStyles={{...textStyles, link:{...textStyles.link, fontFamily:'Metropolis'}}}
```

Comment:
If this is done you should Memoize it.

Reply:
now i memoize textStyles

Reply:
@enahum is there anything left for me to change